light_curves/code_src/ztf_functions.py

```diff
@@ -114,36 +114,31 @@ def locate_objects(coords_list, labels_list, radius):
         journal articles associated with target coordinates, indexed by objectid
     radius : astropy Quantity
         search radius, how far from the source should the archives return results
+     chunksize : int
+        This tap query is much faster when submitting less than ~10,000 coords at a time
+        so iterate over chunks of coords_tbl and then concat results.
 
     Returns
     -------
     locations_df : pd.DataFrame
-        Dataframe with ZTF field, CCD, quadrant and other information that identifies each `coords_list` 
+        Dataframe with ZTF field, CCD, quadrant and other information that identifies each `coords_list`
         object and which parquet files it is in. One row per ZTF objectid.
     """
     # setup for tap query
     tap_service = pyvo.dal.TAPService("https://irsa.ipac.caltech.edu/TAP")
     coords_tbl = make_coordsTable(coords_list, labels_list)
     coordscols = [f"coords.{c}" for c in ["objectid", "label"]]
     ztfcols = [f"ztf.{c}" for c in ["oid", "filtercode", "field", "ccdid", "qid", "ra", "dec"]]
-    query = f"""SELECT {', '.join(coordscols + ztfcols)} 
-        FROM ztf_objects_{DATARELEASE} ztf, TAP_UPLOAD.coords coords 
+    query = f"""SELECT {', '.join(coordscols + ztfcols)}
+        FROM ztf_objects_{DATARELEASE} ztf, TAP_UPLOAD.coords coords
         WHERE CONTAINS(
             POINT('ICRS', coords.ra, coords.dec), CIRCLE('ICRS', ztf.ra, ztf.dec, {radius.value})
         )=1"""
 
-    # this tap query is much faster when submitting less than ~10,000 coords at a time
-    # so iterate over chunks of coords_tbl and then concat results
-    chunksize = 10_000
-    # calculate the number of iterations needed. it would be easier to just use this while loop directly
-    # for the tap calls, but we want to use tqdm and that's easier in a for loop
-    niterations = 0
-    while niterations * chunksize < len(coords_tbl):
-        niterations += 1
     # do the tap calls
     locations = []
-    for i in tqdm(range(niterations)):
-        result = tap_service.run_async(query, uploads={"coords": coords_tbl[i * chunksize : (i + 1) * chunksize]})
+    for i in tqdm.trange(0, len(coords_tbl), chunksize):
```

Comment:
In this function and the one below, 'chunksize' is used to mean two different things.  While we are cleaning up this function, could we please change those variable names to be more descriptive?  Perhaps: "max_number_coords" and "max_number_files"?  I don't love those.

Reply:
I think "chunk" needs to be in there somehow, otherwise it incorrectly implies that some objects/files will not be processed. `max_objects_per_chunk` and `max_files_per_chunk`?

Reply:
If "chunk" seems confusing we could also use "batch". `objects_per_batch` and `files_per_batch`? Could prepend `max_` but could also leave that out of the name and just put those details in the docstring.

Reply:
`chuck` and `chucksize` feel more prevalent in general than batch and the rest. Maybe this is just a documentation question and it should be explained in the docstring? 

Neither of these functions should really be exposed to the user of the notebook.

Either case, I made a note of this, and propose to do the change in the other PR when I also make some of the functions a bit more formally private.

Reply:
> `chuck` and `chucksize` feel more prevalent in general than batch and the rest. Maybe this is just a documentation question and it should be explained in the docstring?

I agree and I always prefer to use the name that the underlying function uses, or some derivative of it. (One of these functions uses a multiprocessing pool which takes the argument "`chunksize`".) And I'm in favor of explaining of the docstring.